### ISSUE TITLE
File to import not found or unreadable: compass

### ISSUE THREAD

**Issue by matheusbaumgart** [madskristensen/WebCompiler#145]
Hello, I'm not quite sure why the compiler is not finding compass since it was working with my previous plugin. Any idea? Does compass works with the plugin? Thanks!

**Comment by madskristensen** [comment]
I haven't tested Compass. This extension uses the `node-sass` component built on top of `libsass` and not the official Ruby compiler.

**Comment by madskristensen** [comment]
@matheusbaumgart I believe compass only works with the Ruby runtime and not the node.js based one. Do you know?

**Comment by matheusbaumgart** [comment]
Yeah, but there is node-compass, that could work nicely. But anyway, I decided to go back to Mindscape, the integration works fine, not as fast as libsass though :/
Thank you anyway.

**Comment by madskristensen** [comment]
I've updated the title

**Comment by wangkanai** [comment]
Has compass feature been added yet?
